Fantasy-Quest/Assets/Scripts/Interaction/Item/IInteractable.cs

```diff
@@ -2,11 +2,7 @@ namespace Interaction.Item
 {
     internal interface IInteractable
     {
-        public bool CanCatInteract
-        {
-            set => CanCatInteract = value;
-            get => CanCatInteract;
-        }
+        public bool CanCatInteract { get; }
```

Comment:
Is it necessary? We should use InteractionByCat() method. And the implementer should think if the action is possible to be done

Reply:
Sometimes it blocks usage for a short period of time, like when the knot is still animating